SpatialGDK/Source/Private/Interop/GlobalStateManager.cpp

```diff
@@ -12,6 +12,10 @@
 #include "SpatialConstants.h"
 #include "Utils/EntityRegistry.h"
 
+DEFINE_LOG_CATEGORY(LogGlobalStateManager);
+
+using namespace improbable;
```

Comment:
Why did we add this, there is nothing in this file(at a quick glance) that requires this.